editoast/src/generated_data/error/switches.rs

```diff
@@ -86,6 +87,17 @@ pub fn check_match_ports_type(
     }
 }
 
+/// Check if node track endpoints are unique
+pub fn check_endpoints_unicity(switch: &ObjectCache, _: &InfraCache, _: &Graph) -> Vec<InfraError> {
```

Comment:
Maybe the function should take a `SwitchCache` directly and let the caller handle correctly the `ObjectCache` unwrapping.

Reply:
We can't do this since this function is added to `OBJECT_GENERATOR` and must follow the generic signature.